src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -69,6 +69,21 @@ services:
             - name: tactician.handler
               command: PrestaShop\PrestaShop\Core\Domain\Product\Query\GetProductIsEnabled
 
+    prestashop.adapter.product.image_thumbnail_provider:
```

Comment:
So we validated this on the last standup, you can remove this thumbnail generator and instead:
- add an `image_type` in the fixtures data called `thumbnail_default` with a size of 45px
- you can add the const type in https://github.com/PrestaShop/PrestaShop/blob/55ffeb752649cc0d1baa0b84b3859d8abf027306/src/Adapter/Product/Image/ProductImagePathFactory.php#L39
- simply adding the image type should automatically generate the image when a new product is created
- maybe you will need to perform a fresh install so that the image is generated during the install process
- we will perform a refacto of these consts in another PR
- we will clean the automatic thumbnail related code in another PR as well
- we will have to handle an upgrade script that generates the missing type for all existing products (in another PR)

Reply:
If all these tasks seem too complicated I can perform them in a separate PR before, so you can use it easily in your PR

Now that I think about it, I believe that's actually the strategy we talked about 😅. So ignore what I said in the previous comment I see you already used the small type for now it's ok!

You will just need to remove this service declaration since the class no longer exists